docs/topics/test-system.md

```diff
@@ -1,72 +1,126 @@
 # Test-system
 
+The test systems are located on a CloudVPS machine at `wikibase-product-testing-2022.wikidata-dev.eqiad1.wikimedia.cloud`.
 
-## Configuration
+You can find all code for running test test systems in the `/opt/test-systems` directory.
+This directory is owned by the `mediawiki` user.
+Each test system is a copy of the `example` docker-compose setup, with customized env vars and settings.
 
-The test-system is currently using a non versioned copy of the [example](../../example/README.md) folder deployed to `wb-product-testing.wikidata-dev.eqiad.wmflabs`.
+Two test systems are maintained, one for the current "latest" major release, and one for the "previous" major release.
 
-Files are located under the `/opt/` folder and for keeping some kind of papertrail a copy of the version to be tested is usually created as `/opt/wmde.1`, `/opt/wmde.2` etc.
+- Latest release (internal ports 83**)
+  - https://wikibase-product-testing.wmcloud.org (8380 internal)
+  - https://wikibase-query-testing.wmcloud.org (8381 internal)
+  - https://wikibase-qs-testing.wmcloud.org (8382 internal)
```

Comment:
Don't seem to be able to initiate Oauth with this

Reply:
Hmm,

So while setting them up I did get the output of:
```
WARNING: The OAUTH_CONSUMER_KEY variable is not set. Defaulting to a blank string.
WARNING: The OAUTH_CONSUMER_SECRET variable is not set. Defaulting to a blank string.
```

But this should all be fine as this is handeled by https://github.com/wmde/wikibase-release-pipeline/blob/main/Docker/build/WikibaseBundle/extra-install/QuickStatements.sh ?

I'll dig around a bit

Reply:
Right, it is probably because I have `QS_PUBLIC_SCHEME_HOST_AND_PORT=https://wikibase-qs-testing-previous.wmflabs.org:8282` and this is likley wrong from the perspective of the code doing oauth.
Also its no longer wmflabs.
I'll make another change and recreate them both

Reply:
> Hmm,
> 
> So while setting them up I did get the output of:
> 
> ```
> WARNING: The OAUTH_CONSUMER_KEY variable is not set. Defaulting to a blank string.
> WARNING: The OAUTH_CONSUMER_SECRET variable is not set. Defaulting to a blank string.
> ```
> 
> But this should all be fine as this is handeled by https://github.com/wmde/wikibase-release-pipeline/blob/main/Docker/build/WikibaseBundle/extra-install/QuickStatements.sh ?
> 
> I'll dig around a bit



> a

Yeah I think this will always be warned about when starting the containers up.

Reply:
Resolved with https://github.com/wmde/wikibase-release-pipeline/pull/315/commits/f9dadd1b0e132a19cef942e79721e8231e34eb04